opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -220,9 +219,10 @@ message Span {
   // Status.Ok (code = 0).
   Status status = 16;
 
-  // An optional number of child spans that were generated while this span
-  // was active. If set, allows an implementation to detect missing child spans.
-  google.protobuf.UInt32Value child_span_count = 17;
+  // An optional number of local child spans that were generated while this span
+  // was active. Value of -1 indicates that the number of local child spans is unknown.
+  // If local_child_span_count>=0, allows an implementation to detect missing child spans.
```

Comment:
nit: You might say if local_child_span_count>0 it allows detecting missing children. The value 0 positively states that there were no local children.

Reply:
I would prefer to treat 0 as “unknown”. It has the same practical result that it can’t be used for validation, but it works better with an optional field. 

Also, it may be worth emphasizing that this only counts in-process children.

Reply:
What if we do know that there is 0 local children? If we treat 0 as unknown we have no way to convey this fact.

Reply:
@yurishkuro I'm merging. Please file an issue if you feel strongly about `0 means unknown`. I understand how it may be confusing - if one simply forgot to set this property to any value - it will mean `0` children which may be factually incorrect

Reply:
@tigrannajaryan we don't need to know, because there is no practical difference. Validation is only useful when this value > 0.

Reply:
https://github.com/open-telemetry/opentelemetry-proto/issues/76